src/custom/state/connection/constants.ts

```diff
@@ -0,0 +1,3 @@
+import { ConnectionType } from 'connection'
+
+export const BACKFILLABLE_WALLETS = [ConnectionType.INJECTED]
```

Comment:
Where is this used?

Reply:
I mean, I saw this `const` added here but did not see it being (re)moved or used anywhere else in the PR